src/pages/ClusterPage/info.js

```diff
@@ -28,6 +28,17 @@ import ShareClusterScreen from './screens/ShareClusterScreen';
 import DeleteClusterModal from './components/DeleteClusterModal';
 import DeploymentStatus from './components/DeploymentStatus';
 import ConnectCluster from './components/ConnectCluster';
+import { ACC_API } from '../../constants/config';
+
+const checkIfUpdateIsAvailable = (image, recipe) => {
+	const version = image.split('/')[1].split(':')[1];
```

Comment:
@lakhansamani Why don't you use `get` from lodash to read the properties which may have `undefined` values.

Reply:
@bietkul that works well for `objects` this is string so here it doesn't make sense, but have added fallback for undefined of first split.